vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -357,6 +357,31 @@ given by the `HierarchyNodes` parameter. This specifies a recursive hierarchy th
         <ReturnType Type="Edm.Boolean" Nullable="false" />
       </Function>
 
+      <Function Name="deentity">
+        <Annotation Term="Core.Description" String="Converts an instance of an entity type into an instance of a complex type with the same structure and values" />
+        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="true" />
+        <ReturnType Type="Edm.ComplexType" Nullable="true" />
+      </Function>
+      <Function Name="deentity">
+        <Annotation Term="Core.Description" String="Converts a collection of instances of an entity type into a collection of instances of a complex type with the same structure and values" />

```

Comment:
Is this a "deep" operation? Assume the `Product` entities have a navigation property `Category`: is that related category also "de-entitied"?

Reply:
No, it is not deep. If you want it deep, you can
```
GET Sales?$apply=
  transformnested(Product,compute(deentity(Entity=Category) as Category1))
  /compute(deentity(Entity=Product) as Product1)
```